include/cinder/app/cocoa/AppImplMac.h

```diff
@@ -87,6 +87,7 @@
   @public
 	AppImplMac*					mAppImpl;
 	NSWindow*					mWin;
+	NSString*					mTitle; // title is cached becasue sometimes we need to restore it after changing window border styles
```

Comment:
Thanks, don't want no body knowing I don't have no good english.  Seriously, the review is appreciated.